clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -87,10 +76,11 @@ class AzureResourceManagerClient extends AzureBaseClient {
     deployment
   }
 
-  ResourceGroup createResouceGroup(AzureCredentials creds, String resourceGroupName, String region) {
-    ResourceGroup rg = new ResourceGroup(region)
+  ResourceGroup createResourceGroup(AzureCredentials creds, String resourceGroupName, String region) {
     try {
-      return this.getResourceManagementClient(creds).getResourceGroupsOperations().createOrUpdate(resourceGroupName, rg).resourceGroup
+      ResourceGroup resourceGroup = this.getResourceManagementClient(creds).getResourceGroupsOperations().createOrUpdate(resourceGroupName,new ResourceGroup(region)).resourceGroup
```

Comment:
in fact, based on feedback from Netflix in previous PR reviews, shouldn't we also remove the "return" statement?